managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -3066,34 +3068,38 @@ boolean shouldCloseLedger(LedgerHandle lh) {
         }
     }
 
-    void switchToNewLedger(final LedgerHandle lh, final VoidCallback callback) {
+    void switchToNewLedger(@Nonnull LedgerHandle lh, VoidCallback callback) {
+        requireNonNull(lh, "LedgerHandle should not be null");
+
         if (log.isDebugEnabled()) {
             log.debug("[{}] Switching cursor {} to ledger {}", ledger.getName(), name, lh.getId());
         }
+
         persistPositionMetaStore(lh.getId(), lastMarkDeleteEntry.newPosition, lastMarkDeleteEntry.properties,
-                new MetaStoreCallback<Void>() {
```

Comment:
Changes here are not necessary. Currently there is no strong code indent rule in the Java code. Please revert these changes.

Reply:
Reverted.